src/components/announcer/CyVerseAnnouncer.js

```diff
@@ -34,6 +41,18 @@ class CyVerseAnnouncer extends Component {
     dequeue = () => {
         if (msgQueue.length > 0) {
             this.setState({ msg: msgQueue.shift(), open: true });
+            if (
+                msgQueue.length === 0 &&
+                this.state.timeout !== EMPTY_QUEUE_TIMEOUT
+            ) {
+                clearInterval(this.state.timer);
+                let timer = setInterval(this.tickCallback, EMPTY_QUEUE_TIMEOUT);
+                this.setState({ timer, timeout: EMPTY_QUEUE_TIMEOUT });
+            } else if (this.state.timeout !== TIMEOUT) {
+                clearInterval(this.state.timer);
+                let timer = setInterval(this.tickCallback, TIMEOUT);
+                this.setState({ timer, timeout: TIMEOUT });
+            }
```

Comment:
Right now we see some delay in displaying the message. This change will display the message right away When the empty queue is empty.